features/step_definitions/api-token.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const Assert = require('chai').assert;
+const jwt = require('jsonwebtoken');
+const request = require('../support/request');
+
+module.exports = function server() {
+    this.Given(/^does not own a token named "([^"]*)"$/, tokenName =>
+        // Ensure there won't be a conflict: delete the token if it's already there
```

Comment:
I suggest refactoring this out somewhere and _also_ do this in an `after` hook for these tests. The expectation should be that the system is in the same "expected" state before and after tests are run.